docs/data-builder-quick-start.md

```diff
@@ -0,0 +1,55 @@
+There are 3 components that need to be added to a project to use 
+Databuilder. These are:
+
+1. Add Data Builder to a `project.yaml`
+2. Add a Dataset definition
+3. Specify a dummy data file. 
+
+### Adding to project.yaml
```

Comment:
suggestion: All the headers on this page should be promoted up one level; this page was taken from a subsection of a page.

```suggestion
## Adding to project.yaml
```